src/website/views/components/data-table/_examples.pug

```diff
@@ -0,0 +1,2712 @@
+          .chi-data-table__cell
+            div 8 Glenlake Lane, San Francisco, California, 94107, USA
+          .chi-data-table__cell
+            div April 8, 2020
+        .chi-data-table__row
+          .chi-data-table__cell.-name.-bold
+            div Ashley Macias
+          .chi-data-table__cell
+            div ashley.macias@domain.com
+          .chi-data-table__cell
+            div 543-723-8733
+          .chi-data-table__cell
+            div 4 W Chapel St, San Jose, California, 95129, USA
+          .chi-data-table__cell
+            div April 8, 2020
+        .chi-data-table__row
+          .chi-data-table__cell.-name.-bold
+            div Cameron Hodge
+          .chi-data-table__cell
+            div cameron.hodge@domain.com
+          .chi-data-table__cell
+            div 509-635-9551
+          .chi-data-table__cell
+            div 7842 Center Rd, San Jose, California, 95129, USA
+          .chi-data-table__cell
+            div April 5, 2020
+      .chi-data-table__footer
+        nav.chi-pagination.-compact(role='navigation', aria-label='Pagination')
+          .chi-pagination__content
+            .chi-pagination__start
+              .chi-pagination__results
+                span.chi-pagination__label 240 Results
+            .chi-pagination__center
+              .chi-pagination__button-group.chi-button-group
+                a(class=`chi-button -icon -flat -sm`, href='#', aria-label='Previous page')
+                  .chi-button__content
+                    i(class="chi-icon icon-chevron-left", aria-hidden='true')
+              .chi-pagination__label
+                strong 2
+                span of
+                strong 3
+              .chi-pagination__button-group.chi-button-group
+                a(class=`chi-button -icon -flat -sm`, href='#', aria-label='Next page')
+                  .chi-button__content
+                    i(class="chi-icon icon-chevron-right", aria-hidden='true')
+            .chi-pagination__end
+
+  .example-tabs.-pl--2
+    ul.chi-tabs#example-base
+      li.-disabled
+        a(href=`#webcomponent-base`) Web Component
+      li.-active
+        a(href=`#html-base`) HTML Blueprint
+  .chi-tabs-panel#webcomponent-base
+  .chi-tabs-panel.-active#html-base
+    :code(lang='html')
+      <div class="chi-data-table -portal -compact">
+      </div>
+
+h3 Portal -- Accordion
+.example.-mb--3
+  .-p--3
+    .chi-data-table.-portal.-compact
+      .chi-data-table__head
+        .chi-data-table__row
+          .chi-data-table__cell.-expandable
+          .chi-data-table__cell
+             span Name
+          .chi-data-table__cell
+            span Email
+          .chi-data-table__cell
+           span Phone Number
+          .chi-data-table__cell
+            span Address
+          .chi-data-table__cell
+            span Last Login
+      .chi-data-table__body
+        .chi-data-table__row.-collapsed
+          .chi-data-table__cell.-expandable
+            i.chi-icon.icon-plus.-xs
```

Comment:
All expand and collapse icons will need to be converted to icon buttons to ensure they're focusable and can be toggled via keyboard.

Example:
```
<button class="chi-button -icon -flat -sm" aria-label="Expand row">
  <div class="chi-button__content">
    <i class="chi-icon icon-plus"></i>
  </div>
</button>
```